### PR TITLE
Improve docs and Fix Quarto Version check

### PR DESCRIPTION
**Pull request by RaymondBalise**
Add better sentence on when to use overwrite = TRUE, set minimal quarto version to be "1.4.549" instead of "2.4.549", add ?rUM page.

### CODE REVIEW

**Review by kyleGrealis — APPROVED** [review]
much clearer feedback for the user. i like it

File: R/make_project.R [hunk 1]
```diff
@@ -65,15 +65,13 @@ make_project <- function(
...
-  
-  
   if (vignette == FALSE){ # make paper project w/o package infrastructure
     # If the project object does not exist add it.
     if (length(list.files(path = path, pattern = "\\.Rproj$")) == 0) {
       usethis::create_project(path = path, open = TRUE, rstudio = TRUE)
     }
   } else { # make paper project with package infrastructure
-    if (type == "Quarto (analysis.qmd)" & the_version < "2.4.549"){
+    if (type == "Quarto (analysis.qmd)" & the_version < "1.4.549"){
```
Comment (kyleGrealis) [inline R/make_project.R:74]: good catch! i'm sorry i missed that earlier

**Review by kyleGrealis — APPROVED** [review]
viewed & approved

File: R/make_project.R (hunk 1 above)
Comment (gabrielodom) [reply R/make_project.R:74]: Please add a comment with URL to the NEWS file or release notes detailing *why* this particular version is the cutoff. Here are some release notes; it's probably in here somewhere: <https://quarto.org/docs/download/release.html>

**Review by gabrielodom — CHANGES_REQUESTED** [review]
We need a descriptive comment on why that particular version of Quarto was chosen.

**Comment by RaymondBalise** [comment]
Where do you want the comment?  The required version was the first to enable quarto for building vignettes.

**Review by gabrielodom — APPROVED** [review]

**Pull request merged**